dexlib2/src/main/java/com/android/tools/smali/dexlib2/dexbacked/DexBackedMethodImplementation.java

```diff
@@ -171,13 +171,11 @@ public Iterator<String> getParameterNames(@Nullable DexReader dexReader) {
     /**
      * Calculate and return the private size of a method implementation.
      *
-     * Calculated as: debug info size + instructions size + try-catch size
+     * Calculated as: instructions size + try-catch size
```

Comment:
Extend the javadoc to say why debug info is not included (it can be shared, and therefore can skew zise calculations).